modules/surge-docs/src/test/scala/docs/command/BankAccountSurgeModel.scala

```diff
@@ -2,16 +2,17 @@
 
 package docs.command
 
-import java.util.UUID
+import com.typesafe.config.ConfigFactory
 
+import java.util.UUID
 import play.api.libs.json.Json
 import surge.core.{ SerializedAggregate, SerializedMessage, SurgeAggregateReadFormatting, SurgeWriteFormatting }
-import surge.internal.commondsl.command.{ AggregateCommandModelBase, SurgeCommandBusinessLogic }
 import surge.kafka.KafkaTopic
+import surge.scaladsl.command.{ AggregateCommandModel, SurgeCommandBusinessLogic }
 
 // #surge_model_class
-object BankAccountSurgeModel extends SurgeCommandBusinessLogic[UUID, BankAccount, BankAccountCommand, BankAccountEvent] {
-  override def commandModel: AggregateCommandModelBase[BankAccount, BankAccountCommand, Nothing, BankAccountEvent] = BankAccountCommandModel
+object BankAccountSurgeModel extends SurgeCommandBusinessLogic[UUID, BankAccount, BankAccountCommand, BankAccountEvent](ConfigFactory.load()) {
```

Comment:
Not that this is a heavy lift to pass in...but would it make sense to default the `conf` in `SurgeCommandBusinessLogic` since probably 90% of the time an app would likely just pass exactly this in